src/core/jsdoc-tsd-parser.ts

```diff
@@ -104,7 +117,7 @@ export class JSDocTsdParser {
 					.trim();
 
 				// ignore everything that is not part of the function description in tsd-files
-				if (cleanedLine && (cleanedLine.startsWith("@param") || cleanedLine.startsWith("@throws") || !cleanedLine.startsWith("@"))) {
+				if (cleanedLine && (cleanedLine.startsWith("@param") || cleanedLine.startsWith("@throws") || cleanedLine.startsWith("@description") || !cleanedLine.startsWith("@"))) {

```

Comment:
You don't have to enable the @description-Tag. Your parsed `jsdocItem` contains an property 'description' which contains only the value which is annotated with "@description". In addition the annotation you can omit this annotation by adding your description on top of the jsdoc comment without any annotations.